faq/design.po

```diff
@@ -60,11 +59,10 @@ msgid ""
 msgstr ""
 "Seule l'instruction ``x++`` sera exécutée si la condition est vraie, mais "
 "l'indentation pourrait vous faire penser le contraire. Mêmes des "
```

Comment:
Le diff causant la fuzzy:
```diff
-indentation leads you to believe otherwise.  Even experienced C programmers will
-sometimes stare at it a long time wondering why ``y`` is being decremented even
+indentation leads many to believe otherwise.  Even experienced C programmers will
+sometimes stare at it a long time wondering as to why ``y`` is being decremented even
```

Je pense que l'intention est bonne, de ne pas "attaquer" le lecteur, on pourrait répercuter ça aussi dans la traduction.

Reply:
J'ai utilisé le diff !